score-client/src/main/resources/application.yml

```diff
@@ -39,12 +39,14 @@ client:
 kfportal:
   url: https://arranger.kids-first.io/june_13/graphql
 
+download:
+  partSize: 250000000
```

Comment:
How did we decide on this changed part size from before?

Reply:
We misunderstood it to different partsize, now it's been reverted to it's original part size.